qemu/tests/cfg/usb.cfg

```diff
@@ -338,6 +338,46 @@
                 - usb_multi_times:
                     no usb_negative_test
                     usb_repeat_times = 20
+        - usb_redir:
+            type = usb_redir
+            only Linux
+            usbredirdev_name = usbredir1
+            chardev_backend_usbredir1 = spicevmc
+            chardev_name_usbredir1 = usbredir
+            display = spice
+            spice_port = 5900
```

Comment:
Suggest `utils_misc.find_free_port`

Reply:
`vm.get_spice_var('spice_port')` and `vm.spice_options.get('spice_port')` all can get the spice port that be generated by the framework.

Reply:
Updated, thanks!